clusterloader2/pkg/config/template_provider.go

```diff
@@ -194,29 +194,45 @@ func validateTestSuite(suite api.TestSuite) error {
 	return nil
 }
 
+func getMappingFromFile(mapping map[string]interface{}, path string) error {
```

Comment:
I don't like the name of this function. Gets suggests that it returns a new map, but this function rather modifies the map in place

Reply:
Indeed, I can see how it can be misleading. How does `updateMappingFromFile` sound?